frontend/src/cards/MapCard.tsx

```diff
@@ -125,14 +128,18 @@ function MapCardWithKey(props: MapCardProps) {
             : exclude(UNKNOWN)
         ),
       /* variableId */ props.variableConfig.variableId,
-      /* timeView */ isCawpCongress ? "cross_sectional" : undefined
+      /* timeView */ isCawp ? "cross_sectional" : undefined
     );
   };
 
+  let countColsToAdd: MetricId[] = [];
+  if (isCawpCongress) countColsToAdd = CAWP_CONGRESS_COUNTS;
+  if (isCawpStateLeg) countColsToAdd = CAWP_STLEG_COUNTS;
+
   const queries = [
     metricQuery(
       Breakdowns.forChildrenFips(props.fips),
-      /* addCountCols */ isCawpCongress
+      /* countColsToAdd */ countColsToAdd
```

Comment:
I dunno; I kind of think when we are using functions that take ordered arguments (rather than like, an object with named properties like react props) it's helpful to outline each argument so it's easy to swap things later / test out different changes, etc. But we don't really have a standard pattern here so im open to everyone's thoughts